extensions/python.bzl

```diff
@@ -144,6 +151,10 @@ This example was a multi-platform set of solutions, pulled into a single
 hub repository named "pip_deps".
 """,
     attrs = {
+        "name": attr.string(
```

Comment:
What is a `tag_class` and it doesn't have a built in `name` attribute?

Reply:
Tag's dont have to produce a repo with any kind of name, they allow a module extension to produce multiple repos